src/models/group.js

```diff
@@ -0,0 +1,26 @@
+"use strict";
+let Sequelize = require("sequelize");
```

Comment:
Consider using "const" for your imports.

Reply:
resolved